### ISSUE TITLE
Need a proper way to implement a window edit menu for windows and linux

### ISSUE THREAD

**Issue by felicienfrancois** [nwjs/nw.js#2802]
On Mac X, there is a [createMacBuiltin](https://github.com/rogerwang/node-webkit/wiki/Menu#menucreatemacbuiltinappname) method to init window menu with the default menu and shortcuts provided by the OS.

On window and linux there is no such API method.
So the best way to create an edit menu with `Copy`, `Cut`, `Paste` and `Select All` is either:
- not to put key/modifiers attribute (so OS shortcuts are not catched), but shortcuts wont be displayed in the menu (as soon as it will work #2153)
- reimplement the functions one by one with something like `document.execCommand("copy");` that will be applied only on root document. If the page contains iframes, all those shortcuts will be broken in its.

In a nutshell there is no proper / fully working way to do such a menu.
So we need either:
- Default menu builder (like createMacBuiltin but for windows or linux)
- Ability to trigger key events on the whole window (including iframes)
- Ability to trigger commands (copy / paste / selectAll / ...) to the whole window (including iframes)
##
